docs/Course Wiki/ECE Course Offerings/ECE313.md

```diff
@@ -0,0 +1,83 @@
+# ECE313
+
+ECE313 (Probability with Engineering Applications) is a 3-credit-hour course that is specifically required for all ECE majors as part of the Technical Core. This is the same as [MATH362](../MATH%20Course%20Offerings/MATH362.md). [STAT410](../Other%20Course%20Offerings/STAT410.md) may substitute this course. ECE313 is offered in the fall, spring, and summer.
```

Comment:
idt we need to link to MATH362 since it's a duplicate course, and if we create a math362 page it would just say to come here